packages/eslint-plugin/docs/rules/enum-const-style.md

```diff
@@ -0,0 +1,61 @@
+# Enforce const enum style (`enum-const-style`)
+
+This rule enforces consistent usage of `const enum`.
+
+## Rule Details
+
+This rule aims to standardize usage of const enums.
+
+## Options
+
+### Default config: `never`
+
+```JSON
+{
+  "enum-const-style": ["error", "never"]
+}
+```
+
+Examples of **incorrect** code for this rule with `never` config:
+
+```ts
+enum Foo {
+  ONE,
+  TWO,
+}
+```
+
+Examples of **correct** code for this rule with `never` config:
+
+```ts
+const enum Foo {
+  ONE,
+  TWO,
+}
+```
+
+### `always` config
+
+Only const enums are allowed
+
+Examples of **incorrect** code for this rule with `always` config:
+
+```ts
+enum const Foo {
+  ONE,
+  TWO,
+}
+```
+
+Examples of **correct** code for this rule with `always` config:
+
+```ts
+const Foo {
+  ONE,
+  TWO,
+}
+```
```

Comment:
I think the examples are the wrong way around

Reply:
actually i haven't read docs, but i did small refactoring in rule source code